src/libcore/hash/mod.rs

```diff
@@ -179,6 +179,13 @@ pub trait Hasher {
     fn write_u64(&mut self, i: u64) {
         self.write(&unsafe { mem::transmute::<_, [u8; 8]>(i) })
     }
+    #[cfg(not(stage0))]
+    /// Write a single `u128` into this hasher.
+    #[inline]
+    #[unstable(feature = "i128", issue = "35118")]
+    fn write_u128(&mut self, i: u128) {
```

Comment:
Isn't there a pathological scenario where another trait also adds the same method and thus code that imports both traits must now resort to UFCS?

(I don't think it matters that in this case one of the args is the new u128 type, since method resolution won't use the signature argument types )